configs/google-vrp/supervisor.conf

```diff
@@ -1,5 +1,6 @@
 [supervisord]
 nodaemon=true
+logfile=/var/log/supervisor/supervisord.log
```

Comment:
I'm thinking this could just be `/dev/stdout`, similar to the application logs below.

Reply:
Looks like that doesn't work. I get the following when trying to run with output to /dev/stdout:
```
2020-11-17 22:53:39,487 CRIT could not write pidfile /supervisord.pid
Traceback (most recent call last):
  File "/usr/bin/supervisord", line 11, in <module>
    load_entry_point('supervisor==3.3.1', 'console_scripts', 'supervisord')()
  File "/usr/lib/python2.7/dist-packages/supervisor/supervisord.py", line 365, in main
    go(options)
  File "/usr/lib/python2.7/dist-packages/supervisor/supervisord.py", line 375, in go
    d.main()
  File "/usr/lib/python2.7/dist-packages/supervisor/supervisord.py", line 84, in main
    self.run()
  File "/usr/lib/python2.7/dist-packages/supervisor/supervisord.py", line 100, in run
    self.options.write_pidfile()
  File "/usr/lib/python2.7/dist-packages/supervisor/options.py", line 1123, in write_pidfile
    self.logger.critical('could not write pidfile %s' % self.pidfile)
  File "/usr/lib/python2.7/dist-packages/supervisor/loggers.py", line 287, in critical
    self.log(LevelsByName.CRIT, msg, **kw)
  File "/usr/lib/python2.7/dist-packages/supervisor/loggers.py", line 293, in log
    handler.emit(record)
  File "/usr/lib/python2.7/dist-packages/supervisor/loggers.py", line 186, in emit
    self.doRollover()
  File "/usr/lib/python2.7/dist-packages/supervisor/loggers.py", line 211, in doRollover
    if not (self.stream.tell() >= self.maxBytes):
IOError: [Errno 29] Illegal seek
```